test/functional/Java11andUp/src/org/openj9/test/java_lang/Test_String.java

```diff
@@ -268,4 +268,59 @@ public void testLines() {
 			// Expected exception
 		}
 	}
+	
+	/*	
+	 * Test Java 12 API String.indent
+	 */
+	// TODO: move to Java12andUp
+	@Test(groups = { "level.sanity" })
+	public void testIndent() {
```

Comment:
There's a lot of duplicated code in your tests. I would make a helper function to improve maintainability.